stackit/internal/services/ske/kubeconfig/resource.go

```diff
@@ -155,6 +157,13 @@ func (r *kubeconfigResource) Schema(_ context.Context, _ resource.SchemaRequest,
 					boolplanmodifier.RequiresReplace(),
 				},
 			},
+			"refresh_before": schema.Int64Attribute{
+				Description: descriptions["refresh_before"],
+				Optional:    true,
+				PlanModifiers: []planmodifier.Int64{
+					int64planmodifier.UseStateForUnknown(),
+				},
+			},
```

Comment:
only a nitpick, but I think it would be great to add here an validator, to avoid that negative numbers will be set
```suggestion
			"refresh_before": schema.Int64Attribute{
				Description: descriptions["refresh_before"],
				Optional:    true,
				PlanModifiers: []planmodifier.Int64{
					int64planmodifier.UseStateForUnknown(),
				},
				Validators: []validator.Int64{
					int64validator.AtLeast(1),
				},
			},
```

Reply:
Sorry, totally missed this. I accepted the suggestion.